apps/vc-api/src/setup.ts

```diff
@@ -33,6 +33,7 @@ async function setupApp(): Promise<INestApplication> {
     type: VersioningType.URI,
     defaultVersion: [API_DEFAULT_VERSION]
   });
+  app.enableShutdownHooks();
```

Comment:
@artursudnik I'm just curious, why this addition?

Reply:
For some reason, the docker container does not want to stop correctly without it. Looks like this implements a good practice of handling signals and exiting a process with signal number + 128 exit code.

Reply:
It could be also implemented with signals handler like this:
```typescript
(signal) => {
    await app.close();
    process.exit(signal + 128);
}
```